scripts/run-integration-tests.sh

```diff
@@ -147,13 +149,38 @@ mkdir -p "$TEST_CONFIG_DIR"
 
 if [[ "$PROVISION" == true ]]; then
     START=$SECONDS
-    #Uncomment Later
-    #up-test-cluster
+#    up-test-cluster
     UP_CLUSTER_DURATION=$((SECONDS - START))
     echo "TIMELINE: Upping test cluster took $UP_CLUSTER_DURATION seconds."
     __cluster_created=1
 fi
 
+
+# TODO: 1. We will run this block of code for Base Version, it will CRDs and push Controller Images to ECR
+# Generate All the CRDs for all the service Base Commit/Tag in /tmp/crd/base
+#ensure_crd_gen "/tmp/crd/base"
+# In general fetch services, build Docker Image for each and push it to ECR if Image:BaseVersion already does not exist.#
+#for d in ./services/*; do
+#    if [ -d "$d" ]; then
+#      echo "$d"
+#        for f in "$d"/*; do
+#          if [ "$f" = "$d"/"Dockerfile" ]; then
+#            echo "$f"
+#            `aws ec2 get-login` returns a docker login string, which we eval here to login to the ECR registry
+# shellcheck disable=SC2046
+#            docker build -t ack/"$d":baseversion .
+#            docker tag "ack"/"$d":baseversion "$AWS_ACCOUNT_ID".dkr.ecr."$AWS_DEFAULT_REGION".amazonaws.com/"$d":baseversion
+#            docker push "$AWS_ACCOUNT_ID".dkr.ecr."$AWS_DEFAULT_REGION".amazonaws.com/"$d"
+#          fi
+#        done
+#    fi
+#done
+
+# TODO: 2. We will run TODO:1 code block for latest version (includes pull request)
+#  generate all CRDs for it to /tmp/crd/test
+#ensure_crd_gen "/tmp/crd/test"
```

Comment:
This needs to be `ensure_controller_gen` and called before we execute `./scripts/generate-crds.sh`